README.md

```diff
@@ -182,7 +182,7 @@ This is open-sourced software licensed under the [MIT License][link_license].
 
 [badge_packagist_version]:https://img.shields.io/packagist/v/avto-dev/back2front-laravel.svg?maxAge=180
 [badge_php_version]:https://img.shields.io/packagist/php-v/avto-dev/back2front-laravel.svg?longCache=true
-[badge_build_status]:https://img.shields.io/github/workflow/status/avto-dev/back2front-laravel/tests/master
+[badge_build_status]:https://img.shields.io/github/actions/workflow/status/avto-dev/back2front-laravel/tests.yml
```

Comment:
@eldario 

Tests [PHP 7.3, basic setup, laravel ^7.0](https://github.com/avto-dev/back2front-laravel/actions/runs/3927578674/jobs/6714593062) was failed, but this is not due to edits in the current PR...